drizzlepac/hlautils/hla_flag_filter.py

```diff
@@ -230,39 +230,41 @@ def ci_filter(drizzled_image, catalog_name, catalog_data, proc_type, param_dict,
     ci_lower_limit = float(param_dict['quality control']['ci filter'][proc_type]['ci_lower_limit'])
     ci_upper_limit = float(param_dict['quality control']['ci filter'][proc_type]['ci_upper_limit'])
     snr = float(param_dict['quality control']['ci filter'][proc_type]['bthresh'])
+par_dict = param_dict['quality control']['ci filter'][proc_type]
+    if param_dict['quality control']['ci filter'][proc_type]['lookup_ci_limits_from_table']:
```

Comment:
```suggestion
    if par_dict['lookup_ci_limits_from_table']:
```
A similar substitution would make the rest of the code a little more readable now that par_dict has been defined.

Reply:
Issue resolved in commit [ab6c352](https://github.com/spacetelescope/drizzlepac/pull/599/commits/ab6c352ff938a5b8dcfe312802da325a2fed94ef)